lib/resty/cassandra/cluster.lua

```diff
@@ -550,6 +551,15 @@ function _Cluster:refresh()
   -- initiate the load balancing policy
   self.lb_policy:init(peers)
 
+  if self.stream_ids == nil then
+    -- Initialize the list of available seed ids (only once)
+    self.stream_ids = {}
```

Comment:
Why not just use a counter for stream_id? It's less computationally expensive and you'll end up writing much less code